README.md

```diff
@@ -111,19 +116,26 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 ad5221e8ae73        ubuntu              "/bin/bash"         36 minutes ago      Up 36 minutes                     		suspicious_mirzakhani
 ```
 
-If this is your directory structure:
+
+In one terminal window, open the snap daemon (in this case with logging set to 1 and trust disabled):
```

Comment:
I believe this should read: `In one terminal window, start the Snap daemon (in this case with logging set to 1 and trust disabled):`

Edits: 
* capitalize Snap if talking about the project
* s/open/start/